docs/man-pages/condor_token_request.rst

```diff
@@ -0,0 +1,143 @@
+    the user ``READ`` authorization, then specifying ``-authz WRITE`` in a
+    token will not allow the user to perform writes.
+ **-debug**
+    Causes debugging information to be sent to ``stderr``, based on the
+    value of the configuration variable ``TOOL_DEBUG``.
+ **-help**
+    Display brief usage information and exit.
+ **-identity** *user@domain*
+    Request a specific identity from the daemon; a client using the resulting token
+    will authenticate as this identity with a remote server.  If not specified, the
+    token will be issued for the ``condor`` identity.
+ **-lifetime** *value*
+    Specify the lifetime, in seconds, for the token to be valid (the
+    token validity will start when the token is signed).  After the
+    lifetime expires, the token cannot be used for authentication.  If
+    not specified, the token will contain no lifetime restrictions.
+ **-name** *hostname*
+    Request a token from the daemon named *hostname* in the pool.  If not specified,
+    the locally-running daemons will be used.
+ **-pool** *pool_name*
+    Request a token from a daemon in a non-default pool *pool_name*.
+ **-token** *filename*
+    Specifies a filename, relative to the directory in the *SEC_TOKEN_DIRECTORY*
+    configuration variable (defaulting to ``~/.condor/tokens.d``), where
+    the resulting token is stored.  If not specified, the token will be
+    sent to ``stdout``.
+ **-type** *type*
+    Request a token from a specific daemon type *type*.  If not given, a
+    *condor_collector* is used.
+
+Examples
+--------
+
+To obtain a token with a lifetime of 10 minutes from the default *condor_collector*
+(the token is not returned until the daemon's administrator takes action):
+
+::
+
+    % condor_token_request -lifetime 600
+    Token request enqueued.  Ask an administrator to please approve request 6108900.
+    eyJhbGciOiJIUzI1NiIsImtpZCI6IlBPT0wifQ.eyJpYX...ii7lAfCA
+
+To request a token from ``bird.cs.wisc.edu`` which is limited to ``READ`` and
+``WRITE``:
+
+::
+
+    % condor_token_request -name bird.cs.wisc.edu \
+                           -identity bucky@cs.wisc.edu
+                           -authz READ -authz WRITE
+    Token request enqueued.  Ask an administrator to please approve request 2578154
+    eyJhbGciOiJIUzI1NiIsImtpZCI6IlBPT0wifQ.eyJpYX...lJTj54
+
+To create a token from the collector in the ``htcondor.cs.wisc.edu`` pool
+and then to save it to ``~/.condor/tokens.d/friend``:
+
+::
+
+    % condor_token_request -pool htcondor.cs.wisc.edu \
+                         -identity friend@cs.wisc.edu \
+                         -lifetime 600 -token friend
+    Token request enqueued.  Ask an administrator to please approve request 2720841.
+
+Exit Status
+-----------
+
+*condor_token_request* will exit with a non-zero status value if it
+fails to request or recieve the token.  Otherwise, it will exit 0.
+
+See also
+--------
+
+:manpage:`condor_token_create(1)`, :manpage:`condor_token_fetch(1)`
```

Comment:
Not going to link to condor_token_request_approve and condor_token_request_auto_approve?

Reply:
I was thinking I'll add those when the respective man pages get written!

Reply:
Fair enough.